docs/09-dev-guide.md

```diff
@@ -51,15 +49,19 @@ mvn clean install -Pquick
 
 ## System Tests
 
-Sleeper's system tests can be used to measure the performance of the standard
-ingest and compaction components of Sleeper. This is useful to ensure that
-performance degradations have not been introduced when we release new versions.
-The system tests load some random data. This allows us to see the number of records
-written per second using the standard ingest process. Once the data has been
-ingested, some compaction jobs will happen. Looking at the logs for these shows
+Sleeper's system tests can be used to measure the performance of the standard ingest and compaction components of Sleeper.
+This is useful to ensure that performance degradations have not been introduced when we release new versions. 
+More information about the performance tests can be found in [12-performance-test.md](12-performance-test.md).
+
+They can also be used to test the functionality of different components, and provide a way to create an instance 
+quickly for testing purposes, which generates random test data for you to work with.
+
+The system tests load some random data. This allows us to see the number of records written per second using the standard 
+ingest process. Once the data has been ingested, some compaction jobs will happen. Looking at the reports shows
 us the number of records per second that a compaction job processes.
```

Comment:
I'm not sure this paragraph is needed any more, with the explanation below of how to use the deployAll test.